cypress/integration/basic-functionalities/local-tests/server/2_Login_tests.spec.js

```diff
@@ -3,93 +3,80 @@ const baseURL = Cypress.env('baseURL')
     })
-  })
 
-  context('Test for logging in with just a password entered', () => {
-    it('should not be able to log in and be remembered', () => {
-      cy.login({ password })
-      cy.url().should('be.equal', baseURL + '/auth/login')
+    context('Test for logging in with just a password entered', () => {
+      it('should not be able to log in and be remembered with just a password entered', () => {
+        cy.login({ password })
+        cy.url().should('be.equal', baseURL + '/auth/login')
+      })
     })
   })
 
-  context('Test for logging in with the already created user and be remembered from server signup test 2', () => {
-    it('should be able to log in and be remembered successfully', () => {
+  context('Test for logging in and be remembered successfully', () => {
+    it('should be able to log in and be remembered successfully when correct password and email are entered', () => {
       cy.login({ email, password })
       // We check if the success toast appears
       cy.get('.border-primary')
     })
   })
+})
 
-  // Tests below starting from here are testing the log in button
-  context('Test for logging in with not registered email', () => {
-    it('should not be able to log in', () => {
+describe('Test for checking the log in functionality', () => {
+  context('Test for logging in with missing data or misconfiguration', () => {
+    it('should not be able to log in with not registered email', () => {
       cy.login({ email: 'email@email.com', password, buttonLoginID: 'button-login' })
       // We check if the error toast appears
       cy.get('[data-test-id="error"]')
     })
-  })
 
-  context('Test for logging in with not registered password', () => {
-    it('should not be able to log in', () => {
+    it('should not be able to log in with not registered password', () => {
       cy.login({ email, password: 'password', buttonLoginID: 'button-login' })
       // We check if the error toast appears
       cy.get('[data-test-id="error"]')
     })
-  })
 
-  context('Test for logging in with no data entered', () => {
-    it('should not be able to log in', () => {
+    it('should not be able to log in with no data entered', () => {
       cy.login({ buttonLoginID: 'button-login' })
       cy.url().should('be.equal', baseURL + '/auth/login')
     })
-  })
 
-  context('Test for logging in with just an email entered', () => {
-    it('should not be able to log in', () => {
+    it('should not be able to log in with just an email entered', () => {
       cy.login({ email, buttonLoginID: 'button-login' })
       cy.url().should('be.equal', baseURL + '/auth/login')
     })
-  })
 
-  context('Test for logging in with just a password entered', () => {
-    it('should not be able to log in', () => {
-      cy.login({ password, buttonLoginID: 'button-login' })
-      cy.url().should('be.equal', baseURL + '/auth/login')
+    context('Test for logging in with just a password entered', () => {
```

Comment:
Why does this need a context()? Also, add a new line before this one so the code can be easier to read.